### ISSUE TITLE
java.io.FileNotFoundException when storing worlds in a different directory

### ISSUE THREAD

**Issue by Kenajcrap** [espidev/ebackup#8]
Pertinent part of server log:
```
[23:12:46 INFO]: [eBackup] Starting backup...
[23:12:46 WARN]: A manual (plugin-induced) save has been detected while server is configured to auto-save. This may affect performance.
[23:12:49 INFO]: [eBackup] Backing up world survival...
[23:12:49 WARN]: java.io.FileNotFoundException: survival (No such file or directory)
[23:12:49 WARN]:        at java.base/java.io.FileInputStream.open0(Native Method)
[23:12:49 WARN]:        at java.base/java.io.FileInputStream.open(FileInputStream.java:219)
[23:12:49 WARN]:        at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
[23:12:49 WARN]:        at dev.espi.ebackup.BackupUtil.zipFile(BackupUtil.java:218)
[23:12:49 WARN]:        at dev.espi.ebackup.BackupUtil.doBackup(BackupUtil.java:106)
[23:12:49 WARN]:        at dev.espi.ebackup.eBackup.lambda$onCommand$1(eBackup.java:144)
[23:12:49 WARN]:        at org.bukkit.craftbukkit.v1_15_R1.scheduler.CraftTask.run(CraftTask.java:84)
[23:12:49 WARN]:        at org.bukkit.craftbukkit.v1_15_R1.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:54)
[23:12:49 WARN]:        at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22)
[23:12:49 WARN]:        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[23:12:49 WARN]:        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[23:12:49 WARN]:        at java.base/java.lang.Thread.run(Thread.java:834)
[23:12:49 INFO]: [eBackup] Backup complete!
```
Started happening after I moved the path to my worlds from the default (root of server dir) to `/server/worlds/` using spigot's --world-dir [startup parameter](https://www.spigotmc.org/wiki/start-up-parameters/?noRedirect=1)

**Comment by espidev** [comment]
will be fixed ASAP

**Comment by espidev** [comment]
Fixed, will be in 1.0.6
